### ISSUE TITLE
Questions on HateBERT_ToxiGen

### ISSUE THREAD

**Issue by youngwook06** [microsoft/TOXIGEN#13]
Hello,

First of all, thank you for your great work!

I have some questions about the details of HateBERT_ToxiGen (https://huggingface.co/tomh/toxigen_hatebert).

1. Is this model the same model used in Table 4. in the paper (ALICE+top-k)?
2. Regarding the description in the paper, "We fine-tune the widely-used HateBERT ... on the **training portion** of TOXIGEN", does the "**training portion**" mean "train subset" with 250,951 samples? or "train split" of "annotated subset" with 8960 samples? (https://huggingface.co/datasets/skg/toxigen-data)

Again, thank you a lot for your great work :)

**Comment by Thartvigsen** [comment]
Thanks for your interest in our work! 

1. The model in this experiment was trained on a balanced subset of ALICE + top-k (about 1700 of each) to study the effects of different generation strategies
2. The released model was trained on the 8960 samples

I'll clarify this in the README, thanks for pointing this out!

**Comment by youngwook06** [comment]
Thanks for your comments! :)

So, the release model (HateBERT_ToxiGen) is trained using **human-annotated** labels for toxicity (8,960 samples) right?

**Comment by Thartvigsen** [comment]
Yes that is right
